docs/development/build-android.md

```diff
@@ -14,37 +14,40 @@ The main non-standard dependency needed for building LiveCode is a Java Developm
 
 ### Installing the Android SDK and NDK
 
-LiveCode requires both the Android Software Development Kit (SDK) and Native Development Kit (NDK).  You can download both from the [Android Developers site](https://developer.android.com/sdk/index.html).
+LiveCode requires both the Android Software Development Kit (SDK) and Native Development Kit (NDK).  You can download both from the [Android Developers site](https://developer.android.com/sdk/index.html). Note that recent versions of the Android SDK are now named "Android Studio command line tools".
```

Comment:
URL for NDK appears to be `https://developer.android.com/ndk/downloads/index.html` and SDK `https://developer.android.com/studio/index.html`